package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "node-firefox-forward-ports",
+  "version": "1.0.0",
+  "description": "Find attached FirefoxOS devices using ADB",
```

Comment:
Huh, interesting: I see connected & attached as synonyms for wired devices. But, if it reads differently to you, I'll change it. 

I used this wording in the README - "Forward local ports to remote debugging interfaces on connected Firefox OS devices"